lib/fastimage/fastimage.rb

```diff
@@ -0,0 +1,405 @@
+class FastImage
+  include FastImageParsing
+
+  VERSION = "2.2.3"
```

Comment:
oh I missed this one when I just released. I think we should just move it to a single file, then you can source it from the gemspec

eg: https://github.com/ruby/rake/blob/master/rake.gemspec#L2-L3